.github/workflows/test-cypress.yml

```diff
@@ -1,10 +1,15 @@
 name: Test (Cypress)
 
-on: [push]
+on:
+  workflow_run:
+    workflows: ["Test (base)"]
+    types:
+      - completed
```

Comment:
They will run on every push because they are triggered by the `test-base` workflow completing, [that runs on every push](https://github.com/chromaui/test-archiver/blob/0a445bf8c14839cd1ba0a7593750392ce949a326/.github/workflows/test-base.yml#L3).

What I want to do is prevent the e2e tests and release workflows from running if the base test job fails. This seems that way to do it, although using `workflow_run` has some weird caveats. Do you know of a better way to do this? Or does it even matter?

Reply:
Let's go with this for now. If we don't like it, we can adjust.

Reply:
@tevanoff what you have done is exactly in line with what I saw a couple weeks ago when seeing how to have the workflows depend on each other. LGTM!